### PR TITLE
V1.01

### PR DESCRIPTION
**Pull request by Mmaxx15**

### CODE REVIEW

**Comment by nyh3** [comment]
Congrats on fixing bug that you caused.

**Comment by Haotian199** [comment]
Congrats you passed 3 checks!

**Comment by HikoHikoHiko** [comment]
Your code looks as clean as my code now! 
Good job on finishing your edit!

File: text-ui-test/EXPECTED.TXT [hunk 1]
```diff
@@ -29,10 +29,10 @@ To List saved expenses, use format:
 To view saved expenses by month, use format:
 << view -m <mm.yyyy> >>
 To view saved expenses by year, use format:
-<< view -y <yyyy> >> 
+<< view -y <yyyy> >>
```
Comment (HikoHikoHiko) [inline text-ui-test/EXPECTED.TXT]: Is this edit necessary?

**Comment by Heart-of-N1** [comment]
Congrats!

File: text-ui-test/EXPECTED.TXT (hunk 1 above)
Comment (Heart-of-N1) [reply text-ui-test/EXPECTED.TXT]: Prof says commit small.
